File: problemas/matematica/2218/2218.jl
using Printf

function somaPA(a1, an, n)
    div((n * (a1 + an)), 2)
end

t = parse(Int, readline())

for _ in 1:t
    n = parse(Int, readline())
    @printf "%d\n" (somaPA(1, n, n) + 1)
end

File: problemas/matematica/2218/2218.cs
using System;

class URI {
    static int somaPA(int a1, int an, int n) {
        return (n * (a1 + an)) / 2;
    }

    static void Main(string[] args) {
        int t = int.Parse(Console.ReadLine());

        for(int i = 0; i < t; ++i){
            int n = int.Parse(Console.ReadLine());
            Console.WriteLine(somaPA(1, n, n) + 1);
        }        
    }
}